lib/util.js

```diff
@@ -26,15 +26,11 @@ function * ensureNonStarterPlan (db) {
 }
 
 function * newTotalExecTimeField (db) {
-  const newTotalExecTimeFieldQuery = 'SELECT current_setting(\'server_version_num\')::numeric >= 130000'
-  const newTotalExecTimeFieldRaw = yield pg.psql.exec(db, newTotalExecTimeFieldQuery)
+  let newTotalExecTimeFieldQuery = `SELECT current_setting('server_version_num')::numeric >= 130000`
+  let newTotalExecTimeFieldRaw = yield pg.psql.exec(db, newTotalExecTimeFieldQuery, ['-t', '-q'])
```

Comment:
Is there a reason these switched to `let`? It doesn't look like these are reassigned, so can we change them back?

```suggestion
  const newTotalExecTimeFieldQuery = `SELECT current_setting('server_version_num')::numeric >= 130000`
  const newTotalExecTimeFieldRaw = yield pg.psql.exec(db, newTotalExecTimeFieldQuery, ['-t', '-q'])
```